force_nevergrad/engine/nevergrad_optimizers.py

```diff
@@ -0,0 +1,330 @@
+import numpy as np
+from functools import partial
+
+from traits.api import (
+    Enum,
+    provides,
+    HasStrictTraits,
+    List
+)
+
+from force_bdss.api import (
+    PositiveInt,
+    FixedMCOParameter,
+    RangedMCOParameter,
+    RangedVectorMCOParameter,
+    ListedMCOParameter,
+    CategoricalMCOParameter,
+    KPISpecification,
+    IOptimizer
+)
+
+import nevergrad as ng
+from nevergrad.functions import MultiobjectiveFunction
+
+
+def translate_mco_to_ng(params):
```

Comment:
Suggestion: these functions (`translate_mco_to_ng`, `translate_ng_to_mco`, `nevergrad_function`) might be better placed in a stand alone `/force_nevergrad/engine/utils.py` file.